### ISSUE TITLE
Add "capture radius" to PanelResizeHandle

### ISSUE THREAD

**Issue by caweidmann** [bvaughn/react-resizable-panels#134]
Thank you for this awesome plugin!

I'm trying to achieve a very specific nested layout with the following behaviour:
- Clean borders (which align perfectly)
- As soon as user comes within certain proximity of a border (e.g. 4px) show the resize handle mouse cursor and allow user to resize panels

Is there some functionality in the plugin to allow for this, or do you have an idea how I can achieve the desired behaviour?

What I have so far is the following:

**Initial layout:** (PanelResizeHandles have a width/height of 9px with a 1px border centered via an additional absolutely positioned div)

<img width="1115" alt="PanelResizeHandle (actual with current implementation)" src="https://user-images.githubusercontent.com/1684894/236803164-1af6cf64-26eb-4cc9-bdf9-5158da9b744f.png">

**On hover of `PanelResizeHandle`:** (I've highlighted them all to show the space the PanelResizeHandles occupy)

<img width="1114" alt="PanelResizeHandle (on hover)" src="https://user-images.githubusercontent.com/1684894/236803191-5fda0d0c-e26f-4e7e-8f0d-a821aca35e1b.png">

**Desired initial layout:**

<img width="1111" alt="PanelResizeHandle (desired initial occupied space)" src="https://user-images.githubusercontent.com/1684894/236803219-22703746-950f-4e2a-80a1-b6ca790c632d.png">

Basically I want to be able to have an element which acts as a `ResizeHandle` but that doesn't occupy any space in the layout and rather sits above the actual `PanelResizeHandle` so when a user comes into contact with the `ResizeHandle` they are actually interacting with the `PanelResizeHandle` which is only 1px in width/height.

**Comment by caweidmann** [comment]
Cool thanks, example worked perfectly.
